### PR TITLE
Align table content with header

### PR DESCRIPTION
**Pull request by wfortin**
Added `mod-align-header` to fit padding with heading padding.

Extract variable in file headers to a single `variables.scss`, this will allow easier use of "piece by piece" files and override our default values easily.

### CODE REVIEW

File: scss/tables/table.scss [hunk 1]
```diff
@@ -30,6 +27,18 @@ table {
...
 
+  &.mod-align-header {
+    th, td {
+      &:first-child {
+        padding-left: $header-padding;
+      }
+    }
+
+    tbody tr.selected td:first-child {
+      padding-left: 35px;
```
Comment (marilynechan) [inline scss/tables/table.scss]: This number surely comes from `$header-padding - [selected-border-width]`?

**Comment by marilynechan** [comment]
Nice cleanup! :ship: :it:

File: scss/tables/table.scss (hunk 1 above)
Comment (wfortin) [inline scss/tables/table.scss]: Good point, fixed it.

**Comment by GermainBergeron** [comment]
LGTM!

**Pull request merged**